vite.config.mjs

```diff
@@ -68,6 +68,31 @@ export default defineConfig((config) => {
           entryFileNames: 'assets/[name].[hash:22].js',
           chunkFileNames: 'assets/[name].[hash:22].js',
           assetFileNames: 'assets/[name].[hash:22][extname]',
+          manualChunks: {
+            vendor_react: ['react', 'react-dom', 'react/jsx-runtime'],
+            vendor_react_router: [
+              'react-router',
+              'react-router-dom',
+              'react-router-dom-v5-compat',
+            ],
+            vendor_date_fns: ['date-fns'],
+            vendor_lodash: [
+              'lodash/get',
+              'lodash/isEqual',
+              'lodash/isNull',
+              'lodash/isNil',
+              'lodash/isEmpty',
+              'lodash/isString',
+              'lodash/isNumber',
+              'lodash/isBoolean',
+              'lodash/isUndefined',
+              'lodash/isArray',
+              'lodash/gt',
+              'lodash/omit',
+              'lodash/flatMap',
+              'lodash/groupBy',
+            ],
```

Comment:
would we be able to leverage the use of `['lodash']` from https://rollupjs.org/configuration-options/#output-manualchunks so that we don't need to list each individual function one by one?

Reply:
I attempted that, however it did not work and included everything. I think mainly because we're not using `@rollup/plugin-node-resolve` we're using whatever is built into Vite

Reply:
thanks for checking

Reply:
Wanted to double double check, tried using that plugin and it did not work, so we're stuck with this 😢